app/experimenter/nimbus-ui/src/components/GQLErrorAlert/index.tsx

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ApolloError, ServerError, ServerParseError } from "@apollo/client";
+import { GraphQLError } from "graphql";
+import React from "react";
+import { Alert } from "react-bootstrap";
+import sentryMetrics from "../../services/sentry";
+
+type NetworkError = ApolloError["networkError"];
+
+// Type guards - If the `networkError` is a `ServerError`, we want to display its `result.errors`
+// array containing extra info
+function isNetworkServerError(
+  networkError: NetworkError,
+): networkError is ServerError {
+  return (networkError as ServerError)?.result !== undefined;
+}
+
+// If the `networkError` is a `ServerParseError`, we want to display `bodyText` and `statusCode`
+function isNetworkServerParseError(
+  networkError: NetworkError,
+): networkError is ServerParseError {
+  return (
+    (networkError as ServerParseError)?.bodyText !== undefined &&
+    (networkError as ServerParseError)?.statusCode !== undefined
+  );
+}
+
+export const GQLErrorAlert = ({ error }: { error: ApolloError }) => {
+  // without this, we only have access to error.message
+  const allErrors = JSON.parse(JSON.stringify(error));
```

Comment:
This was a bit awkward, I got it from [this comment](https://github.com/apollographql/apollo-client/issues/1496#issuecomment-290303638).

Seems like an alternative is to `.catch` the error after every query and deconstruct `networkError`/`graphQLErrors`/`error` there and then send those in instead, but this seemed slightly preferable to me.